backend/backend/buy_house.py

```diff
@@ -0,0 +1,42 @@
+"""Module enables players to purchase a
+ house on properties of a given name"""
+import sys
+import json
+import cgitb
+
+from backend.player import Player
+from backend.properties import Property, get_position_by_name
+from backend.game import get_games
+
+cgitb.enable()
+
+
+def add_house(source=sys.stdin, output=sys.stdout):
+    """Adds a house to a property.
+    """
+    output.write('Content-Type: application/json\n\n')
+    request = json.load(source)
+    player_id = request["player_id"]
+    property_name = request["property_name"]
+    property_position = get_position_by_name(player_id, property_name)
+    position = next(iter(property_position.values()))
+    games = get_games()
+    houses = None
+
+    with Player(player_id) as player:
+        for game in games:
+            if player.username in games[game]:
+                game_id = game
+                break
+        with Property(position, game_id) as prop:
+            if prop.houses < 4 and prop.hotels == 0:
+                prop.houses += 1
+                houses = prop.houses
+                player.balance -= prop.house_price
+            else:
+                prop.houses = 0
+                prop.hotels = 1
+                houses = 5
+                player.balance -= prop.house_price
```

Comment:
I've fixed this in the wrong branch. Will be fixed in my other branch when merged. Don't mind for the moment!